generator.rb

```diff
@@ -0,0 +1,73 @@
+def is_vowel?(letter)
+  vowels = ["a", "e", "i", "o", "u"]
+  if vowels.include?(letter.downcase)
+    return true
+  end
+  return false
+end
+
+
+def get_word_from_user(word)
+  puts "What is the #{word} to combine?"
+  word = gets.chomp.to_s
+  while word.length < 2 || word.include?(" ")
+    puts "Word must be at least 2 characters long, and must not contain any spaces."
+    word = gets.chomp.to_s
+  end
+  return word
+end
+
+
+def chop_first_word(first_word)
+  stop_index = first_word.length - 1
+
+  first_word.reverse.each_char do |letter|
+    if is_vowel?(letter)
+      return first_word[0...stop_index]
+    end
+    stop_index -= 1
+  end
+  stop_index = first_word.length
+  return first_word[0...stop_index]
+end
+
+
+def chop_second_word(second_word)
```

Comment:
I bet you could find a clever way to combine `chop_first_word` and `chop_second_word`